.infra/index.ts

```diff
@@ -306,6 +306,9 @@ if (isAdhocEnv) {
         { targetPort: 3000, port: 80, name: 'http' },
         { targetPort: 9464, port: 9464, name: 'metrics' },
       ],
+      backendConfig: {
+        customRequestHeaders: ['X-Client-Region:{client_region}'],
+      },
```

Comment:
Is `x-client-region` blocked from req headers, eg. clients can't inject it from req headers?

Reply:
haven't tested it but i think gcp lb overrides it no matter what. we can try